github/issues.go

```diff
@@ -77,13 +77,14 @@ func (i Issue) IsPullRequest() bool {
 // It is separate from Issue above because otherwise Labels
 // and Assignee fail to serialize to the correct JSON.
 type IssueRequest struct {
-	Title     *string   `json:"title,omitempty"`
-	Body      *string   `json:"body,omitempty"`
-	Labels    *[]string `json:"labels,omitempty"`
-	Assignee  *string   `json:"assignee,omitempty"`
-	State     *string   `json:"state,omitempty"`
-	Milestone *int      `json:"milestone,omitempty"`
-	Assignees *[]string `json:"assignees,omitempty"`
+	Title       *string   `json:"title,omitempty"`
+	Body        *string   `json:"body,omitempty"`
+	Labels      *[]string `json:"labels,omitempty"`
+	Assignee    *string   `json:"assignee,omitempty"`
+	State       *string   `json:"state,omitempty"`
+	StateReason *string   `json:"state_reason,omitempty"`
```

Comment:
Thank you, @parkr - let's go ahead and add a comment here so that people don't have to jump to the official docs as much:

```go
// StateReason can be 'completed' or 'not_planned'.
```

Reply:
Added!

Reply:
Perfect, thank you!